inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/stream/InlongStreamServiceImpl.java

```diff
@@ -385,6 +424,16 @@ public PageResult<InlongStreamInfo> listAll(InlongStreamPageRequest request) {
         return pageResult;
     }
 
+    private void unpackExtParams(InlongStreamInfo streamInfo) {
+        String extParams = streamInfo.getExtParams();
```

Comment:
It seems that there is a method with the same function in line 244. It is recommended to unify the two methods

Reply:
ok, done